### PR TITLE
Update docs to work exclusively with bower

### PR DESCRIPTION
**Pull request by robdodson**
- Kill git submodules
- Move all apps to their own appspot domains

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -21,11 +21,15 @@
...
   "dependencies": {
-    "smoothscroll": "~0.0.2",
+    "smoothscroll": "^0.0.2",
     "x-tag-imports": "x-tag/x-tag-imports",
     "marked": "*",
     "highlightjs": "*",
-    "plunker-button": "robdodson/plunker-button#^0.4.1",
-    "topeka": "Polymer/topeka"
+    "plunker-button": "robdodson/plunker-button#^0.5.0",
+    "core-elements": "Polymer/core-elements#^0.5.2",
```
Comment (robdodson) [inline bower.json]: Sure!

File: docs/elements/material.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ and scrolling effects.
 For more detail on the material design philosophy and guidelines, see the
 [Material design specification](http://google.com/design/spec).
 
-For a sample of the material design patterns in use, see the <a href="/apps/topeka/" target="_blank">Topeka sample app</a>.
+For a sample of the material design patterns in use, see the <a href="http://polymer-topeka.appspot.com" target="_blank">Topeka sample app</a>.
```
Comment (robdodson) [inline docs/elements/material.md]: wow I had no idea. that's really cool! I'll update

**Comment by ebidel** [comment]
:+1:

**Comment by arthurevans** [comment]
LGTM.

**Pull request merged**